### PR TITLE
[de] Add "Fantasyserie(n)" to spelling and POS

### PR DESCRIPTION
**Pull request by Discostu36**

### CODE REVIEW

File: languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/added.txt [hunk 1]
```diff
@@ -37324,4 +37324,12 @@ Mythosschwarzs	Mythosschwarz	SUB:GEN:SIN:NEU
 Felsgrau	Felsgrau	SUB:AKK:SIN:NEU
 Felsgrau	Felsgrau	SUB:DAT:SIN:NEU
 Felsgrau	Felsgrau	SUB:NOM:SIN:NEU
-Felsgraus	Felsgrau	SUB:GEN:SIN:NEU
\ No newline at end of file
+Felsgraus	Felsgrau	SUB:GEN:SIN:NEU
+Fantasyserie    Fantasyserie  SUB:AKK:SIN:FEM
```
Comment (danielnaber) [inline languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/added.txt:37328]: You need to use tabs here, not spaces.

Comment (Discostu36) [reply languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/added.txt:37328]: Ah, sorry, seems like my editor converted the tabs to spaces. Will fix tonight.

Comment (Discostu36) [reply languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/added.txt:37328]: should be fixed now

**Comment by Discostu36** [comment]
I just resolved the conflicts, so this would be a good point for merging :-)

**Comment by Discostu36** [comment]
This has been manually added by others in the meanwhile.

**Pull request closed without merging**